app/_plugins/tags/tabs/template.erb

```diff
@@ -17,6 +17,10 @@
   </ul>
 
   <div class="tabs-component-panels">
+    <div class="meshservice">
+      I am using new <a href="">MeshService</a> <input type="checkbox">
```

Comment:
```suggestion
      I am using <a href="">MeshService</a> <input type="checkbox">
```

Stay away from `new` because what's new one day doesn't stay new for long :)